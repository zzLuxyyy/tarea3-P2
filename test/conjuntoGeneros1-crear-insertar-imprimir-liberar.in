crearConjuntoGeneros 4
imprimirConjuntoGeneros
liberarConjuntoGeneros
crearConjuntoGeneros 2
insertarConjuntoGeneros 1
insertarConjuntoGeneros 5
insertarConjuntoGeneros 1
imprimirConjuntoGeneros
insertarConjuntoGeneros 0
imprimirConjuntoGeneros
liberarConjuntoGeneros
Fin