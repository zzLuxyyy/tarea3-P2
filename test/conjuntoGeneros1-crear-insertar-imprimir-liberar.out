1>Se creó conjunto generos con cantMax 4 exitosamente.
2>Conjunto de generos:

3>Se ejecutó 'liberarTConjuntoGeneros' exitosamente.
4>Se creó conjunto generos con cantMax 2 exitosamente.
5>Se ejecutó 'insertarTConjuntoGeneros con genero de id 1 exitosamente.
6>Se ejecutó 'insertarTConjuntoGeneros con genero de id 5 exitosamente.
7>Se ejecutó 'insertarTConjuntoGeneros con genero de id 1 exitosamente.
8>Conjunto de generos:
1
9>Se ejecutó 'insertarTConjuntoGeneros con genero de id 0 exitosamente.
10>Conjunto de generos:
0 1
11>Se ejecutó 'liberarTConjuntoGeneros' exitosamente.
12>Fin.
